pkg/informer/informer.go

```diff
@@ -44,6 +75,162 @@ func InformerForResource(res schema.GroupVersionResource) (GenericInformer, erro
 	return i, nil
 }
 
+func isResourceAllowed(cm *v1.ConfigMap, group, kind string, allowedList []Resource, deniedList []Resource) bool {
```

Comment:
I don't think we use `cm` in this function.
Also cm and allowList/denyList contain redundant information so let's not require both.